### ISSUE TITLE
time parameter

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-azurerm#646]
_This issue was originally opened by @LaneEnglot as hashicorp/terraform#16950. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version

```
.terraform --version
Terraform v0.11.1
+ provider.azurerm v0.3.1
```

### Terraform Configuration Files

```
resource "azurerm_servicebus_queue" "k8s_dev1_sb_que" {
  name                = "k8s-dev1-sb-que"
  resource_group_name = "${azurerm_resource_group.k8s-dev1-rg.name}"
  namespace_name      = "${azurerm_servicebus_namespace.k8s_dev1_sb.name}"
  requires_duplicate_detection = "False"
  default_message_ttl = "00:30:00"
  duplicate_detection_history_time_window = "00:15:00"
  enable_partitioning = true
}

resource "azurerm_servicebus_topic" "k8s_dev1_sb_topic" {
  name                = "k8s-dev1-sb-topic"
  resource_group_name = "${azurerm_resource_group.k8s-dev1-rg.name}"
  namespace_name      = "${azurerm_servicebus_namespace.k8s_dev1_sb.name}"
  enable_partitioning = true
  default_message_ttl = "00:30:00"
  duplicate_detection_history_time_window = "00:15:00"
}
```

### Output
```
* azurerm_servicebus_queue.k8s_dev1_sb_que: servicebus.QueuesClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="BadRequest" Message="The string '00:30:00' is not a valid TimeSpan value. CorrelationId: 35d09bb5-cca0-4d3f-a456-cd99a2e3db73"
* azurerm_servicebus_topic.k8s_ukbeta_sb_topic: 1 error(s) occurred:

* azurerm_servicebus_topic.k8s_dev1_sb_topic: servicebus.TopicsClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="BadRequest" Message="The string '00:30:00' is not a valid TimeSpan value. CorrelationId: 45dc663c-d549-47fb-8c09-10083abf7e2d"
* azurerm_servicebus_queue.k8s_uat_sb_que: 1 error(s) occurred:
```

### Expected Behavior
Terraform should have created the que and topic with the correct ttl values...

### Actual Behavior
This worked in in .0.10.x, I upgraded and it will not accept the values for TTL.

### Steps to Reproduce
Please list the full steps required to reproduce the issue, for example:
1. `terraform init`
2. `terraform apply`

**Comment by sjwaight** [comment]
Some samples of the values that can be used for the TimeSpan fields also wouldn't be missed in the provider documentation.

**Comment by steve-hawkins** [comment]
@tombuildsstuff, apologies if I've missed something, but I'm a little lost on this issue and why the bug label has been changed to documentation?

I have been trying with what I believe are valid TimeSpan formats and still get the following error:-

`The string '7.00:00:00' is not a valid TimeSpan value.`

If this is a documentation issue could you point me at the accepted TimeSpan formats?

**Comment by barabasmate** [comment]
This format seems to be working: P7DT0H0M0.0S 
This is 7days.

**Comment by tombuildsstuff** [comment]
Fixed via #1921
